src/filters.rs

```diff
@@ -197,3 +200,38 @@ impl ResolveFilter for LowerFilter {
         Ok(content)
     }
 }
+
+#[derive(Debug)]
+pub struct SlugifyFilter;
+
+impl ResolveFilter for SlugifyFilter {
+    fn resolve<'t, 'py>(
+        &self,
+        variable: Option<Content<'t, 'py>>,
+        py: Python<'py>,
+        _template: TemplateString<'t>,
+        _context: &mut Context,
+    ) -> TemplateResult<'t, 'py> {
+        let non_word_re = Regex::new(r"[^\w\s-]").unwrap();
+        let whitespace_re = Regex::new(r"[-\s]+").unwrap();
```

Comment:
I know that you shouldn't be using unwrap in production code in general. Though, in this case this should be safe, as the input will never change. 

I also realized that we can't really use constants for regex at the top of the module which is a common pattern in python. I did see that there is a crate `lazy-static` that would help you with that, but perhaps it seems a bit overkill to include another dependency for that.

Reply:
For cases like this, I'd go with `.expect` to explain why it won't panic.

Reply:
As for lazy-static, I understand that's considered obsolete now. Instead we can use something like [`LazyLock`](https://doc.rust-lang.org/stable/std/sync/struct.LazyLock.html).